source/whats-new.txt

```diff
@@ -32,6 +33,21 @@ Learn what's new in:
 
 .. _upcoming-breaking-changes:
 
+.. _csharp-version-2.29:
+
+What's New in 2.29
+------------------
+
+The 2.29 driver release has been tested against {+mdb-server+} version 8.0 and
+includes the following new features:
```

Comment:
S: this phrasing doesn't fully explain whether the version is compatible or partially compatible
maybe something like this is clearer:
```suggestion
The 2.29 driver release is fully compatible with {+mdb-server+} version 8.0 and
includes the following new features:
```

Reply:
According to the release notes, there are some 8.0 features that aren't compatible. Can update this wording to reflect that.